lib/pka_dev.h

```diff
@@ -164,45 +164,63 @@ typedef struct
     uint8_t        res_cnt;
 } pka_dev_gbl_shim_res_info_t;
 
+struct pka_dev_mem_res
+{
+    uint64_t eip154_base;         ///< base address for eip154 mmio registers
+    uint64_t eip154_size;         ///< eip154 mmio register region size
+
+    uint32_t wndw_ram_off_mask;   ///< common offset mask for alt window ram and window ram

```

Comment:
shouldn't be unit64_t? Note that a mask should have the same length as the value that would be applied to

Reply:
Yes, agreed. thanks